docs/mma.md

```diff
@@ -144,6 +144,16 @@ User Attributes
     <td>A boolean indicating that the user has at least one search widget placed on the home screen</td>
     <td><a href="https://github.com/mozilla-mobile/fenix/pull/4694#issuecomment-520591275">#4694</a></td>
   </tr>
+  <tr>
+    <td>`tracking_protection_enabled`</td>
```

Comment:
Would we want “changed” instead of enabled? Do we care about the disabled case?

Reply:
@eliserichards - we requested for 'enabled' to be used so that we can understand if someone has enabled or disabled protection settings. This could be used for future feature discovery messaging.

Reply:
:+1: